src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -136,6 +137,12 @@ public static bool IsNonZeroLowerBoundArraySupported
 
         public static bool IsOpenSslSupported => IsLinux || IsFreeBSD || Isillumos || IsSolaris;
 
+#if BUILT_WITH_EAT
+        public static bool IsBuiltWithAggressiveTrimming => true;
```

Comment:
This is a trimming setting so you can set to false by default and change it to true with a linker substitution